### PR TITLE
add more table in meta dump

### PR DESCRIPTION
**Pull request by pengweisong**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

#### Description:
Add the newly added tables in meta when dump.

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by critical27 — COMMENTED** [review]
Good job, since you replace the literal string prefix with `MetaKeyUtils`, maybe you could replace all of them~

**Comment by pengweisong** [comment]
@critical27 The rest prefix did not have corresponding prefix function in MetaKeyUtils.cpp. Is it OK to add extra functions for this? If it's OK, I'll add them.

**Comment by critical27** [comment]
> @critical27 The rest prefix did not have corresponding prefix function in MetaKeyUtils.cpp. Is it OK to add extra functions for this? If it's OK, I'll add them.

Ah, I understand. It is ok to me

File: src/tools/meta-dump/MetaDumpTool.cpp [hunk 1]
```diff
@@ -52,8 +54,9 @@ class MetaDumper {
       }
     }
     {
-      LOG(INFO) << "Partition info";
-      prefix = "__parts__";
+      LOG(INFO) << "------------------------------------------\n\n";
+      LOG(INFO) << "Partition info::";
```
Comment (darionyaphet) [inline src/tools/meta-dump/MetaDumpTool.cpp:58]: why double colon?

Comment (pengweisong) [reply src/tools/meta-dump/MetaDumpTool.cpp:58]: It will let the split boundary more obvious.

**Review by critical27 — APPROVED** [review]

**Comment by critical27** [comment]
Good job

File: src/tools/meta-dump/MetaDumpTool.cpp [hunk 2]
```diff
@@ -87,8 +103,56 @@ class MetaDumper {
...
+      iter->Seek(rocksdb::Slice(prefix));
+      while (iter->Valid() && iter->key().starts_with(prefix)) {
+        auto key = folly::StringPiece(iter->key().data(), iter->key().size());
+        auto val = folly::StringPiece(iter->value().data(), iter->value().size());
+        auto addr = MetaKeyUtils::parseHostDirKey(key);
+        auto dir = MetaKeyUtils::parseHostDir(val);
+
+        std::string dataDirs = "";
+        for (auto d : dir.get_data()) {
+          dataDirs += d + ", ";
```
Comment (SuperYoko) [inline src/tools/meta-dump/MetaDumpTool.cpp:118]: Maybe `dataDirs += d + ", ";` is better like following code.

**Review by SuperYoko — APPROVED** [review]

File: src/tools/meta-dump/MetaDumpTool.cpp (hunk 2 above)
Comment (SuperYoko) [reply src/tools/meta-dump/MetaDumpTool.cpp:118]: And this is okay for me.

**Comment by HarrisChu** [comment]
just curious, could we move "__meta_version__" to table, and also dump it?

**Comment by pengweisong** [comment]
@HarrisChu meta version is indeed in meta table, I could add it if you need.

**Review by SuperYoko — APPROVED** [review]

**Review by critical27 — APPROVED** [review]
Good job

**Pull request merged**